apps/api/src/app/environments-v1/e2e/environments.controller.e2e.ts

```diff
@@ -36,6 +36,9 @@ describe('Env Controller', async () => {
       });
     });
   });
+
+
+
```

Comment:
@cursoragent let's also add a test case to check the new protection for renaming